etna/ciim/client.py

```diff
@@ -212,7 +231,9 @@ def search(
             "sortOrder": sort_order,
             "template": template,
             "aggregations": format_list_param(aggregations),
-            "filterAggregations": format_list_param(filter_aggregations),
+            "filterAggregations": format_list_param(
+                filter_aggregations, "filter_aggregations"
+            ),
```

Comment:
Hi @TNA-Allan. As per the conversation on https://national-archives.atlassian.net/browse/DF-238, we want to remove the `format_list_param` function completely and just let `requests` handle the list of values as per the example at end of https://docs.python-requests.org/en/latest/user/quickstart/#passing-parameters-in-urls

Reply:
There are issues found with the API response and is recorded in the same ticket. Hence created draft PR not to be merged.